Assets/RedBlueGames/MulliganRenamer/Editor/MulliganRenamerPreviewPanel.cs

```diff
@@ -295,16 +295,22 @@ private static void ApplyBackgroundColorToWhitespaces(Rect rect, GUIStyle style,
                 {
                     var blockRect = new Rect(position.x + totalRect.x - style.padding.left, position.y, 0, totalRect.y);
                     var spaceBlocks = GetConsecutiveBlocksOfToken(coloredText.Text, ' ');
+
                     foreach (var block in spaceBlocks)
                     {
                         var blockSize = blockStyle.CalcSize(new GUIContent(block));
 
                         blockRect.width = blockSize.x;
+                        var textureWidth = ((blockRect.x + blockRect.width) < (rect.x + rect.width))
```

Comment:
This is good, but 2 small things.
1 - you can replace all rect.x + rect.width with rect.xMax.
2 - It still hangs over the edge a few pixels on the 2nd column (but not the 1st). I couldn't immediately figure out why. Basically there's some reason the second column (and not the first column) xMax actually extends past the divider.
```
var textureWidth = ((blockRect.xMax) < (rect.xMax))
                            ? blockRect.width
                            : Mathf.Max(0f, rect.xMax - blockRect.x);
```

Reply:
![image](https://user-images.githubusercontent.com/398377/71832240-8abbe580-3078-11ea-9af2-34589d2ab326.png)

Another hint with a closer look - the text in the first column cuts off a bit earlier than the background rect. So it's possible the column rects aren't quite right. It might have to do with the hotspot padding I added last night, so that I could shrink the dividers.

Reply:
I think we need to subtract from the width in line 218:

`
secondColumnRect.x += firstColumnRect.width + DividerWidth * 2;
secondColumnRect.width = style.SecondColumnWidth - DividerWidth * 2;
`

This will fix the rect size, and by doing so, we also fix the highlight problem.

Reply:
Yep seems right - except maybe not * 2? The 2 is to account for the width of the divider on the left and the right? I dunno, tried it with 1 and still seemed ok.

Reply:
ahm... ok haha then, 1 it is!
I am not currently in my computer, so there is no way to test, sorry about that.

Reply:
NP. Looks good, I'll just merge and fix it up here then.

Reply:
Great! thanks